ExampleReduction.ipynb

```diff
@@ -168,26 +169,13 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "# DO NOT RUN ME IF YOU HAVE ALREADY REDUCED THE DATA BEFORE\n",
```

Comment:
It depends. You might have to repopulate the DB.

Reply:
Will change the comment.